### PR TITLE
fix #32574 sephora.com

### LINKED ISSUE #32574: sephora.com
### Issue URL (Incorrect Blocking)
[https://www.sephora.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.sephora.com%2F)
### Comment
Username: @DreDri
> Problem: Unable to sign in. Typically the front page (shown) has the menu drop down and any of the selections lead to a pop-up to log in. 
> Note: For whatever reason, when AdGuard is turned off, I don't get the typical pop-up, I get a full login screen (shown). I'm not sure if this is helpful - or if it tells you maybe Sephora's site has changed? This is definitely AdGuard because, as you will see, I'm able to sign in when it's turned off. 
> 
> Thank you for all your tireless work. Yours is definitely a thankless job. I think most people are grateful, they just suck at expressing it.

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.0.11
Browser: | Chrome
Filters: | AdGuard Mobile Ads,<br>AdGuard Annoyances,<br>AdGuard Base,<br>EasyList,<br>EasyPrivacy,<br>Fanboy's Annoyances,<br>Adblock Warning Removal List,<br>(Obsolete) Anti-Adblock Killer | Reek,<br>AdGuard Tracking Protection,<br>AdGuard Social Media,<br>Peter Lowe's list,<br>Fanboy's Enhanced Tracking List
Other extensions used: | AdBlock (thats the whole name... red stop sign with a white hand up = icon)

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/32574

### CODE REVIEW

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1064,6 +1064,7 @@ svyaznoy.ru##.b-footer-subscribe-block
 svyaznoy.ru##a[href="/personal/subscription/go"]
 flagandcross.com##.essb_mailform
 flagandcross.com##.essbasc-popup
+sephora.com##div[data-comp="EmailSignUp"]
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/annoyances.txt]: Not shown in code, but on mobile this newsletter:

##.newsletter-signup

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/annoyances.txt (hunk 1 above)
Comment (zloyden) [reply AnnoyancesFilter/sections/annoyances.txt]: In mobile version on my side same data-comp.
Did you noticed popup (mobile-app) `data-comp="AndroidAppBanner StickyBanner Box"` in mobile? I reproduce it right now.
<details><summary>Screenshot:</summary>

Снимок экрана 2019-07-24 в 16 16 38

</details><br/>

Comment (BlazDT) [reply AnnoyancesFilter/sections/annoyances.txt]: Was not showing for me.

Comment (Alex-302) [reply AnnoyancesFilter/sections/annoyances.txt]: don't hide in the footer

Comment (zloyden) [reply AnnoyancesFilter/sections/annoyances.txt]: ok

**Pull request closed without merging**